src/rules/each-key-value-single-line/README.md

```diff
@@ -0,0 +1,48 @@
+# each-key-value-single-line
+
+ Disallow situations where users loop over map-keys and fetch the value inside the loop.
```

Comment:
Is there some way yo make the description a bit clearer? Might be a bit confusing for some users.

Reply:
Could be even this:

```
This is a rule that checks for situations where users have:

 - Done a loop using map-keys
 - Grabbed the value for that key inside of the loop.

```